api/v1/ptpoperatorconfig_types.go

```diff
@@ -68,11 +68,15 @@ type PtpEventConfig struct {
 	// +kubebuilder:default=false
 	// EnableEventPublisher will deploy event proxy as a sidecar
 	EnableEventPublisher bool `json:"enableEventPublisher,omitempty"`
-	// TransportHost format is <protocol>://<transport-service>.<namespace>.svc.cluster.local:<transport-port>"
-	// Example HTTP transport: "http://ptp-event-publisher-service.openshift-ptp.svc.cluster.local:9043"
+	// TransportHost format is <protocol>://<transport-service>-<nodename>.<namespace>.svc.cluster.local:<transport-port>
+	// Example HTTP transport: "http://ptp-event-publisher-service.openshift-ptp-NODE_NAME.svc.cluster.local:9043"
 	// Example AMQP transport: "amqp://amq-router-service-name.amq-namespace.svc.cluster.local"
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Transport Host",xDescriptors={"urn:alm:descriptor:com.tectonic.ui:text"}
 	TransportHost string `json:"transportHost,omitempty"`
+	// +kubebuilder:default=emptyDir
+	// TransportStorageType format is emptyDir for ephemeral storage or name of StorageClass for persist storage
+	//+operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Transport Storage Type",xDescriptors={"urn:alm:descriptor:com.tectonic.ui:text"}
+	TransportStorageType string `json:"transportStorageType,omitempty"`
```

Comment:
just call storageType, don't use transport

Reply:
can you list down allowed storage types values here

Reply:
done